<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemarkDlg</class>
 <widget class="QDialog" name="RemarkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>人员</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="user_comboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>注释</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="3">
    <widget class="QPlainTextEdit" name="remark_plainTextEdit"/>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>192</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="date_Label">
     <property name="text">
      <string>date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>141</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" rowspan="2">
    <widget class="QPushButton" name="oky_btn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" rowspan="2">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>撤销</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
